<launch>
  <include file="/home/rpn2/catkin_ws/src/rpn/stage/nav/lab/move_base_config/move_base.xml"/>

  <node name="map_server" pkg="map_server" type="map_server" args="/home/rpn2/catkin_ws/src/rpn/stage/nav/lab/map.pgm 0.05" respawn="false" >
  </node>
  <node pkg="stage_ros" type="stageros" name="stageros" args="-g /home/rpn2/catkin_ws/src/rpn/stage/nav/lab/lab.world" respawn="false" >
  <remap from="/clock" to="clock" />
  </node>
<node pkg="rpn" type="robot_pose.py" name="robot_pose" />

<node pkg="amcl" type="amcl" name="amcl" respawn="true">
  <remap from="scan" to="base_scan" />
  <param name="initial_pose_x" value="4.6"/>
  <param name="initial_pose_y" value="4.5"/>
  <param name="odom_model_type" value="omni"/>
</node>

</launch>
